nav2_rotation_shim_controller/src/nav2_rotation_shim_controller.cpp

```diff
@@ -73,6 +73,8 @@ void RotationShimController::configure(
     node, plugin_name_ + ".rotate_to_goal_heading", rclcpp::ParameterValue(false));
   nav2_util::declare_parameter_if_not_declared(
     node, plugin_name_ + ".rotate_to_heading_once", rclcpp::ParameterValue(false));
+  nav2_util::declare_parameter_if_not_declared(
+    node, plugin_name_ + ".feedback", rclcpp::ParameterValue(std::string("CLOSED_LOOP")));
```

Comment:
You could make this simply a bool for `closed_loop = true` default and avoid the string parsing :-) There's only 2 options